packages/app-shell/src/components/Modal/index.jsx

```diff
@@ -88,10 +94,14 @@ const Modal = ({ modal, openModal }) => {
       openModal(MODALS.trialExpired)
     }
 
+    //Check if Pendo loads on the page - we don't want to show the OB Migration modal if there is a Pendo guide already visible
+    const isPendoModalVisible = window.pendo && !window.pendo.isGuideShown() || !window.pendo ? false : true;
```

Comment:
```suggestion
    const isPendoModalVisible = window.pendo && !window.pendo.isGuideShown() ? false : true;
```
I believe that we don't need the extra OR condition. Could it be that I'm missing something?

Reply:
Is there any reason to go for the negation? Aren't all the cases included if we just use this:
`const isPendoModalVisible = window.pendo && window.pendo.isGuideShown();`

Reply:
@gisete, do you need to cover any extra scenario or could `const isPendoModalVisible= window.pendo && window.pendo.isGuideShown()` be enough here?

Reply:
I had deleted the comment by mistake, re-added it here ☝️ 😂   When creating boolean variables, you often don't need ternary conditions returning true and false, I usually either change the negative conditions to positive or use the double negation (eg. !!window.pendo instead of !window.pendo ? false : true )

Reply:
Hey @anafilipadealmeida and @dinostheo, I admit it's a bit confusing 😄   My thinking here was that `!window.pendo` covers when we're working locally and Pendo doesn't load and `window.pendo && !window.pendo.isGuideShown()` check if Pendo loads and a Pendo modal is up . But feel free to suggest another approach if it makes more sense.

Reply:
If all you need to know is if the Pendo modal is up, we know that the modal will only be shown if pendo exists and isGuideShown is true. Then you can confidently say that in any other scenario the modal is not there (so no need to think about local vs production, as you are just focusing on pendo being defined and modal being up). By using this: `const isPendoModalVisible= window.pendo && window.pendo.isGuideShown()` in theory you are covering all the scenarios. Is this making any sense @gisete ?

Reply:
You're totally right @anafilipadealmeida ! I think I got confused with another method I was using before to check if Pendo was available. I'll update it. Thanks for the feedback 🙂